uaclient/tests/test_security_status.py

```diff
@@ -189,7 +192,8 @@ def test_get_ua_info(self, m_status, is_attached, FakeConfig):
             ),
             (
                 mock_version(
-                    "1.0", [MOCK_ORIGINS["archive_main"], MOCK_ORIGINS["now"]]
+                    "1.0",
+                    [MOCK_ORIGINS["archive_main"], MOCK_ORIGINS["now"]],
```

Comment:
`black` is insisting on these changes - I guess we haven't run `black` on this file since we bumped the version?

Reply:
Nah we run `black` on everything :black_heart: 
This is one of the few things `black` is not strongly opinionated on: the trailing comma. If you remove the comma (as it was before) it can stay as is, but if there is a comma, then you need to split lines.

Reply:
now I see there are many instances of it - which version of `black` is doing this?

Reply:
oh trailing commas!! Okay I'll try removing them. I dislike the unnecessary diff